docker/base_images/base_image.Dockerfile.jinja

```diff
@@ -23,4 +23,6 @@
     FROM python:{{python_version}}
 {% endif %}
 
-{% include truss_build_template_path %}
+{% with use_gpu=use_gpu, python_version=python_version, live_reload=live_reload %}
```

Comment:
should we just revert this to what it was before the last PR since we ended up just inlining what we need in the other base.Dockerfile.jinja